  INFO root [2019-03-02 09:58:15,724] - 
  INFO root [2019-03-02 09:58:15,725] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 09:58:15,726] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 09:58:15,727] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 09:58:15,727] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 09:58:15,757] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:13446 )
  INFO root [2019-03-02 09:58:15,757] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:13446, SOMAXCONN=5.
  INFO root [2019-03-02 09:58:15,757] - server-configs:
  INFO root [2019-03-02 09:58:15,757] - 	internalTcpAddr : 192.168.0.148:13446
  INFO root [2019-03-02 09:58:15,757] - 	componentID : 6000
  WARN root [2019-03-02 09:58:15,760] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 09:58:15,800] - Components::findLogger: found logger, addr:192.168.0.148:13264
  INFO root [2019-03-02 09:58:15,896] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 09:58:15,896] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 09:58:15,920] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 3532 ----
  INFO root [2019-03-02 09:58:15,920] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 09:58:15,920] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 09:58:15,920] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 09:58:16,238] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 09:58:16,238] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 09:58:22,349] - Forwarding logs to logger[192.168.0.148:13264/0/0/0]...
  INFO root [2019-03-02 10:02:21,416] - 
  INFO root [2019-03-02 10:02:21,416] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:02:21,416] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:02:21,416] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:02:21,416] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:02:21,432] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:1726 )
  INFO root [2019-03-02 10:02:21,432] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:1726, SOMAXCONN=5.
  INFO root [2019-03-02 10:02:21,432] - server-configs:
  INFO root [2019-03-02 10:02:21,432] - 	internalTcpAddr : 192.168.0.148:1726
  INFO root [2019-03-02 10:02:21,432] - 	componentID : 6000
  WARN root [2019-03-02 10:02:21,432] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:02:21,510] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 8140 ----
  INFO root [2019-03-02 10:02:21,510] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:02:21,510] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:02:21,510] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:02:21,947] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 10:02:21,947] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:02:29,292] - Components::findComponents: find logger(1)...
  WARN root [2019-03-02 10:02:29,776] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20539, errorno=10048
  INFO root [2019-03-02 10:02:30,792] - Components::findComponents: find logger(2)...
  WARN root [2019-03-02 10:02:31,870] - Components::findComponents: not found logger!
  INFO root [2019-03-02 10:02:32,885] - Components::findComponents: find dbmgr(1)...
  INFO root [2019-03-02 10:02:34,963] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:02:37,041] - Components::findComponents: find dbmgr(3)...
  INFO root [2019-03-02 10:02:39,088] - Components::findComponents: find dbmgr(4)...
  INFO root [2019-03-02 10:02:41,166] - Components::findComponents: find dbmgr(5)...
  INFO root [2019-03-02 10:02:43,244] - Components::findComponents: find dbmgr(6)...
  INFO root [2019-03-02 10:02:45,322] - Components::findComponents: find dbmgr(7)...
  INFO root [2019-03-02 10:02:47,369] - Components::findComponents: find dbmgr(8)...
  INFO root [2019-03-02 10:02:49,449] - Components::findComponents: find dbmgr(9)...
  INFO root [2019-03-02 10:02:51,527] - Components::findComponents: find dbmgr(10)...
  INFO root [2019-03-02 10:02:53,605] - Components::findComponents: find dbmgr(11)...
  INFO root [2019-03-02 10:02:55,683] - Components::findComponents: find dbmgr(12)...
  INFO root [2019-03-02 10:02:57,761] - Components::findComponents: find dbmgr(13)...
  INFO root [2019-03-02 10:02:59,839] - Components::findComponents: find dbmgr(14)...
  INFO root [2019-03-02 10:03:01,932] - Components::findComponents: find dbmgr(15)...
  INFO root [2019-03-02 10:03:03,995] - Components::findComponents: find dbmgr(16)...
  WARN root [2019-03-02 10:03:06,073] - Components::findComponents: find dbmgr(17)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:08,138] - Components::findComponents: find dbmgr(18)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:10,232] - Components::findComponents: find dbmgr(19)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:12,310] - Components::findComponents: find dbmgr(20)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:14,387] - Components::findComponents: find dbmgr(21)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:16,465] - Components::findComponents: find dbmgr(22)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:18,543] - Components::findComponents: find dbmgr(23)...
delay time is too long, please check the dbmgr logs!
  WARN root [2019-03-02 10:03:20,621] - Components::findComponents: find dbmgr(24)...
delay time is too long, please check the dbmgr logs!
  INFO root [2019-03-02 10:03:39,026] - 
  INFO root [2019-03-02 10:03:39,042] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:03:39,042] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:03:39,042] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:03:39,042] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:03:39,058] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:1761 )
  INFO root [2019-03-02 10:03:39,058] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:1761, SOMAXCONN=5.
  INFO root [2019-03-02 10:03:39,058] - server-configs:
  INFO root [2019-03-02 10:03:39,058] - 	internalTcpAddr : 192.168.0.148:1761
  INFO root [2019-03-02 10:03:39,058] - 	componentID : 6000
  WARN root [2019-03-02 10:03:39,058] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:03:40,573] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 11676 ----
  INFO root [2019-03-02 10:03:40,573] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:03:40,573] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:03:40,573] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:03:40,995] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 10:03:40,995] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 10:03:41,495] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20547, errorno=10048
  INFO root [2019-03-02 10:03:48,872] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:03:48,934] - Components::findComponents: found logger, addr:192.168.0.148:1763
  INFO root [2019-03-02 10:03:48,934] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:03:48,934] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:03:48,950] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:03:49,059] - Components::findComponents: found dbmgr, addr:192.168.0.148:1768
  INFO root [2019-03-02 10:03:49,059] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:03:49,059] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:03:49,059] - Components::findComponents: found dbmgr, addr:192.168.0.148:1768
  WARN root [2019-03-02 10:03:50,075] - Forwarding logs to logger[192.168.0.148:1763/0/0/0]...
 ERROR root [2019-03-02 10:09:01,281] - Components::removeComponentByChannel: baseappmgr : 5000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

 ERROR root [2019-03-02 10:09:01,306] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:09:01,310] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:09:01,310] - ==> [2019-03-02 10:09:01 265] TCPPacketReceiver::processPendingEvents(192.168.0.148:1766): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:09:01,311] - ==> [2019-03-02 10:09:01 282] Components::removeComponentByChannel: baseappmgr : 5000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:09:01,311] - ==> [2019-03-02 10:09:01 303] TCPPacketReceiver::processPendingEvents(192.168.0.148:1763): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:09:01,311] - ==> [2019-03-02 10:09:01 307] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:09:01,311] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1768): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:09:01,312] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:09:01,333] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:09:01,334] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:09:01,334] - Shutdowner::onShutdownEnd: shutting down
  WARN root [2019-03-02 10:09:01,334] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1791): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-03-02 10:09:01,334] - Components::removeComponentByChannel: baseapp : 7001, Normal exit!
  WARN root [2019-03-02 10:09:01,334] - TCPPacketReceiver::processPendingEvents(192.168.0.148:1793): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  WARN root [2019-03-02 10:09:01,335] - Cellappmgr::removeCellapp: erase cellapp[8001], currsize=0
  INFO root [2019-03-02 10:09:01,335] - Components::removeComponentByChannel: cellapp : 8001, Normal exit!
 DEBUG root [2019-03-02 10:09:01,336] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-03-02 10:09:01,659] - ThreadPool::destroy(): waiting for thread(2)[0x143f708,0x143f498,], try=1
 DEBUG root [2019-03-02 10:09:02,061] - TPThread::~TPThread(): 0x143f708
 DEBUG root [2019-03-02 10:09:02,061] - TPThread::~TPThread(): 0x143f498
 DEBUG root [2019-03-02 10:09:02,061] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-03-02 10:09:02,099] - Bundle::destroyObjPool(): size 4.
 DEBUG root [2019-03-02 10:09:02,099] - Channel::destroyObjPool(): size 16.
 DEBUG root [2019-03-02 10:09:02,100] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 10:09:02,100] - UDPPacket::destroyObjPool(): size 3.
 DEBUG root [2019-03-02 10:09:02,100] - EndPoint::destroyObjPool(): size 16.
 DEBUG root [2019-03-02 10:09:02,100] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 10:09:02,100] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 10:09:02,100] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-03-02 10:09:02,100] - cellappmgr(6000) has shut down.
  INFO root [2019-03-02 10:09:03,113] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2019-03-02 10:10:34,495] - 
  INFO root [2019-03-02 10:10:34,495] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:10:34,495] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:10:34,495] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:10:34,495] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:10:34,502] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2712 )
  INFO root [2019-03-02 10:10:34,502] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2712, SOMAXCONN=5.
  INFO root [2019-03-02 10:10:34,502] - server-configs:
  INFO root [2019-03-02 10:10:34,502] - 	internalTcpAddr : 192.168.0.123:2712
  INFO root [2019-03-02 10:10:34,502] - 	componentID : 6000
  WARN root [2019-03-02 10:10:34,504] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:10:34,620] - Components::findLogger: found logger, addr:192.168.0.123:2683
  INFO root [2019-03-02 10:10:34,620] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:10:34,620] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:10:34,622] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 3972 ----
  INFO root [2019-03-02 10:10:34,622] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:10:34,622] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:10:34,622] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:10:34,972] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 10:10:34,972] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 10:10:41,169] - Forwarding logs to logger[192.168.0.123:2683/0/0/0]...
 ERROR root [2019-03-02 10:13:34,109] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:13:34,109] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:13:34,109] - ==> [2019-03-02 10:13:34 112] TCPPacketReceiver::processPendingEvents(192.168.0.123:2683): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:13:34,109] - ==> [2019-03-02 10:13:34 113] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:13:34,139] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2685): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:13:34,139] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:13:34,139] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:13:34,149] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:13:34,149] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:13:34,149] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:16:41,242] - 
  INFO root [2019-03-02 10:16:41,242] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:16:41,252] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:16:41,252] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:16:41,252] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:16:41,259] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2852 )
  INFO root [2019-03-02 10:16:41,259] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2852, SOMAXCONN=5.
  INFO root [2019-03-02 10:16:41,259] - server-configs:
  INFO root [2019-03-02 10:16:41,259] - 	internalTcpAddr : 192.168.0.123:2852
  INFO root [2019-03-02 10:16:41,259] - 	componentID : 6000
  WARN root [2019-03-02 10:16:41,263] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  WARN root [2019-03-02 10:16:41,889] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20483, errorno=10048
  INFO root [2019-03-02 10:16:41,890] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 11368 ----
  INFO root [2019-03-02 10:16:41,890] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:16:41,890] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:16:41,890] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:16:42,223] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 10:16:42,223] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:16:49,489] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:16:49,510] - Components::findComponents: found logger, addr:192.168.0.123:2851
  INFO root [2019-03-02 10:16:49,510] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:16:49,511] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:16:49,512] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:16:49,614] - Components::findComponents: found dbmgr, addr:192.168.0.123:2850
  INFO root [2019-03-02 10:16:49,614] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:16:49,615] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:16:49,615] - Components::findComponents: found dbmgr, addr:192.168.0.123:2850
  WARN root [2019-03-02 10:16:50,615] - Forwarding logs to logger[192.168.0.123:2851/0/0/0]...
 ERROR root [2019-03-02 10:17:07,851] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:17:07,851] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:17:07,851] - ==> [2019-03-02 10:17:07 854] TCPPacketReceiver::processPendingEvents(192.168.0.123:2851): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:17:07,851] - ==> [2019-03-02 10:17:07 855] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:17:07,878] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2850): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:17:07,878] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:17:07,888] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:17:07,888] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:17:07,888] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:17:07,888] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:17:15,662] - 
  INFO root [2019-03-02 10:17:15,662] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:17:15,669] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:17:15,669] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:17:15,669] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:17:15,672] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2897 )
  INFO root [2019-03-02 10:17:15,672] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2897, SOMAXCONN=5.
  INFO root [2019-03-02 10:17:15,672] - server-configs:
  INFO root [2019-03-02 10:17:15,672] - 	internalTcpAddr : 192.168.0.123:2897
  INFO root [2019-03-02 10:17:15,672] - 	componentID : 6000
  WARN root [2019-03-02 10:17:15,672] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:17:17,182] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 10456 ----
  INFO root [2019-03-02 10:17:17,182] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:17:17,182] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:17:17,182] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:17:17,609] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 10:17:17,609] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:17:24,759] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:17:24,843] - Components::findComponents: found logger, addr:192.168.0.123:2899
  INFO root [2019-03-02 10:17:24,844] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:17:24,844] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:17:24,845] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:17:24,948] - Components::findComponents: found dbmgr, addr:192.168.0.123:2902
  INFO root [2019-03-02 10:17:24,949] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:17:24,949] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:17:24,950] - Components::findComponents: found dbmgr, addr:192.168.0.123:2902
  WARN root [2019-03-02 10:17:25,950] - Forwarding logs to logger[192.168.0.123:2899/0/0/0]...
 ERROR root [2019-03-02 10:18:33,693] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:18:33,695] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:18:33,695] - ==> [2019-03-02 10:18:33 694] TCPPacketReceiver::processPendingEvents(192.168.0.123:2899): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:18:33,695] - ==> [2019-03-02 10:18:33 695] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:18:33,731] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2902): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:18:33,732] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:18:33,739] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:18:33,742] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:18:33,743] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:18:33,743] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:20:36,632] - 
  INFO root [2019-03-02 10:20:36,632] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:20:36,632] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:20:36,632] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:20:36,632] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:20:36,638] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:2994 )
  INFO root [2019-03-02 10:20:36,638] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:2994, SOMAXCONN=5.
  INFO root [2019-03-02 10:20:36,638] - server-configs:
  INFO root [2019-03-02 10:20:36,638] - 	internalTcpAddr : 192.168.0.123:2994
  INFO root [2019-03-02 10:20:36,638] - 	componentID : 6000
  WARN root [2019-03-02 10:20:36,638] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:20:38,150] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 7716 ----
  INFO root [2019-03-02 10:20:38,150] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:20:38,150] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:20:38,150] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:20:38,568] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 10:20:38,572] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:20:45,888] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:20:45,928] - Components::findComponents: found logger, addr:192.168.0.123:2993
  INFO root [2019-03-02 10:20:45,928] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:20:45,928] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:20:45,928] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:20:46,048] - Components::findComponents: found dbmgr, addr:192.168.0.123:2988
  INFO root [2019-03-02 10:20:46,048] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:20:46,048] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:20:46,048] - Components::findComponents: found dbmgr, addr:192.168.0.123:2988
  WARN root [2019-03-02 10:20:47,067] - Forwarding logs to logger[192.168.0.123:2993/0/0/0]...
 ERROR root [2019-03-02 10:27:41,787] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:27:41,787] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:27:41,787] - ==> [2019-03-02 10:27:41 794] TCPPacketReceiver::processPendingEvents(192.168.0.123:2993): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:27:41,787] - ==> [2019-03-02 10:27:41 795] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:27:41,817] - TCPPacketReceiver::processPendingEvents(192.168.0.123:2988): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:27:41,817] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:27:41,827] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:27:41,827] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:27:41,827] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:27:41,827] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:35:36,386] - 
  INFO root [2019-03-02 10:35:36,390] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:35:36,390] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:35:36,390] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:35:36,390] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:35:36,407] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:3250 )
  INFO root [2019-03-02 10:35:36,407] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:3250, SOMAXCONN=5.
  INFO root [2019-03-02 10:35:36,407] - server-configs:
  INFO root [2019-03-02 10:35:36,407] - 	internalTcpAddr : 192.168.0.123:3250
  INFO root [2019-03-02 10:35:36,407] - 	componentID : 6000
  WARN root [2019-03-02 10:35:36,414] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  WARN root [2019-03-02 10:35:37,020] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20589, errorno=10048
  INFO root [2019-03-02 10:35:37,022] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 4024 ----
  INFO root [2019-03-02 10:35:37,022] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:35:37,022] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:35:37,022] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:35:37,409] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 10:35:37,409] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:35:44,675] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:35:44,737] - Components::findComponents: found logger, addr:192.168.0.123:3254
  INFO root [2019-03-02 10:35:44,738] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:35:44,738] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:35:44,739] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:35:44,841] - Components::findComponents: found dbmgr, addr:192.168.0.123:3251
  INFO root [2019-03-02 10:35:44,841] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:35:44,842] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:35:44,842] - Components::findComponents: found dbmgr, addr:192.168.0.123:3251
  WARN root [2019-03-02 10:35:45,843] - Forwarding logs to logger[192.168.0.123:3254/0/0/0]...
 ERROR root [2019-03-02 10:46:59,094] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:46:59,094] - The following logs sent to logger failed:
  WARN root [2019-03-02 10:46:59,094] - ==> [2019-03-02 10:46:59 099] TCPPacketReceiver::processPendingEvents(192.168.0.123:3254): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:46:59,094] - ==> [2019-03-02 10:46:59 100] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 10:46:59,135] - TCPPacketReceiver::processPendingEvents(192.168.0.123:3251): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 10:46:59,144] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 10:46:59,144] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 10:46:59,144] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 10:46:59,144] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 10:46:59,144] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 10:48:41,817] - 
  INFO root [2019-03-02 10:48:41,817] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 10:48:41,817] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 10:48:41,817] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 10:48:41,817] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 10:48:41,825] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.123 ( 192.168.0.123:3604 )
  INFO root [2019-03-02 10:48:41,825] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.123:3604, SOMAXCONN=5.
  INFO root [2019-03-02 10:48:41,825] - server-configs:
  INFO root [2019-03-02 10:48:41,825] - 	internalTcpAddr : 192.168.0.123:3604
  INFO root [2019-03-02 10:48:41,825] - 	componentID : 6000
  WARN root [2019-03-02 10:48:41,825] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 10:48:43,344] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 11376 ----
  INFO root [2019-03-02 10:48:43,344] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 10:48:43,344] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 10:48:43,344] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 10:48:43,724] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 10:48:43,724] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 10:48:50,885] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 10:48:50,954] - Components::findComponents: found logger, addr:192.168.0.123:3603
  INFO root [2019-03-02 10:48:50,954] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:48:50,954] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:48:50,955] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 10:48:50,961] - Components::findComponents: found logger, addr:192.168.0.123:3603
  INFO root [2019-03-02 10:48:50,963] - Components::findComponents: found dbmgr, addr:192.168.0.123:3606
  INFO root [2019-03-02 10:48:50,964] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 10:48:50,964] - Components::addComponent[dbmgr], uid=28936, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 10:48:51,065] - Components::findComponents: found dbmgr, addr:192.168.0.123:3606
  WARN root [2019-03-02 10:48:52,065] - Forwarding logs to logger[192.168.0.123:3603/0/0/0]...
 ERROR root [2019-03-02 11:24:21,582] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 11:24:21,610] - The following logs sent to logger failed:
  WARN root [2019-03-02 11:24:21,613] - ==> [2019-03-02 11:24:21 570] TCPPacketReceiver::processPendingEvents(192.168.0.123:3603): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 11:24:21,614] - ==> [2019-03-02 11:24:21 583] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 11:24:21,648] - TCPPacketReceiver::processPendingEvents(192.168.0.123:3606): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 11:24:21,654] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 11:24:21,660] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 11:24:21,662] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 11:24:21,662] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-03-02 11:24:21,663] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-03-02 14:44:38,432] - 
  INFO root [2019-03-02 14:44:38,445] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 14:44:38,486] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 14:44:38,487] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 14:44:38,487] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 14:44:38,524] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:13637 )
  INFO root [2019-03-02 14:44:38,524] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:13637, SOMAXCONN=5.
  INFO root [2019-03-02 14:44:38,524] - server-configs:
  INFO root [2019-03-02 14:44:38,525] - 	internalTcpAddr : 192.168.0.148:13637
  INFO root [2019-03-02 14:44:38,525] - 	componentID : 6000
  WARN root [2019-03-02 14:44:38,554] - invalid UID(0) <= 0, please check UID for environment! automatically set to 28936.
  INFO root [2019-03-02 14:44:38,675] - Components::findLogger: found logger, addr:192.168.0.148:13513
  INFO root [2019-03-02 14:44:38,676] - Components::addComponent: reset orderLog, uid=28936!
  INFO root [2019-03-02 14:44:38,676] - Components::addComponent[logger], uid=28936, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 14:44:38,700] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 28936. PID: 14976 ----
  INFO root [2019-03-02 14:44:38,700] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 14:44:38,701] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 14:44:38,702] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 14:44:39,361] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 14:44:39,362] - Components::process(): Request for the process of identity...
  WARN root [2019-03-02 14:44:45,573] - Forwarding logs to logger[192.168.0.148:13513/0/0/0]...
 ERROR root [2019-03-02 14:45:30,914] - Components::removeComponentByChannel: cellapp : 8001, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

 ERROR root [2019-03-02 14:45:30,926] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 14:45:30,928] - The following logs sent to logger failed:
  WARN root [2019-03-02 14:45:30,979] - ==> [2019-03-02 14:45:30 927] TCPPacketReceiver::processPendingEvents(192.168.0.148:13513): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 14:45:30,980] - ==> [2019-03-02 14:45:30 928] Components::removeComponentByChannel: logger : 2000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  WARN root [2019-03-02 14:45:30,980] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13649): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  WARN root [2019-03-02 14:45:30,982] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13566): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-03-02 14:45:30,982] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

  INFO root [2019-03-02 14:45:30,995] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-03-02 14:45:30,995] - Shutdowner::onShutdown: shutting down
  INFO root [2019-03-02 14:45:30,996] - Shutdowner::onShutdownEnd: shutting down
  WARN root [2019-03-02 14:45:30,996] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13606): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-03-02 14:45:30,997] - Components::removeComponentByChannel: baseappmgr : 5000, Normal exit!
  WARN root [2019-03-02 14:45:30,997] - TCPPacketReceiver::processPendingEvents(192.168.0.148:13737): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
  INFO root [2019-03-02 14:45:30,998] - Components::removeComponentByChannel: baseapp : 7001, Normal exit!
 DEBUG root [2019-03-02 14:45:31,028] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-03-02 14:45:31,337] - ThreadPool::destroy(): waiting for thread(2)[0x143d868,0x143da70,], try=1
 DEBUG root [2019-03-02 14:45:31,738] - TPThread::~TPThread(): 0x143d868
 DEBUG root [2019-03-02 14:45:31,738] - TPThread::~TPThread(): 0x143da70
 DEBUG root [2019-03-02 14:45:31,738] - ThreadPool::destroy(): successfully!
 DEBUG root [2019-03-02 14:45:31,783] - Bundle::destroyObjPool(): size 17.
 DEBUG root [2019-03-02 14:45:31,784] - Channel::destroyObjPool(): size 16.
 DEBUG root [2019-03-02 14:45:31,784] - TCPPacket::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 14:45:31,784] - UDPPacket::destroyObjPool(): size 6.
 DEBUG root [2019-03-02 14:45:31,785] - EndPoint::destroyObjPool(): size 16.
 DEBUG root [2019-03-02 14:45:31,785] - Address::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 14:45:31,785] - TCPPacketReceiver::destroyObjPool(): size 0.
 DEBUG root [2019-03-02 14:45:31,785] - UDPPacketReceiver::destroyObjPool(): size 0.
  INFO root [2019-03-02 14:45:31,785] - cellappmgr(6000) has shut down.
  INFO root [2019-03-02 14:45:32,787] - EventDispatcher()::~EventDispatcher: Num timers = 1
  INFO root [2019-03-02 14:45:57,857] - 
  INFO root [2019-03-02 14:45:57,876] - -----------------------------------------------------------------------------------------


  INFO root [2019-03-02 14:45:57,877] - Resmgr::initialize: KBE_ROOT=D:/kbengine-2.4.2/
  INFO root [2019-03-02 14:45:57,879] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine-2.4.2/kbe/res/;D:/kbengine-2.4.2/first_server_assets/;D:/kbengine-2.4.2/first_server_assets/scripts/;D:/kbengine-2.4.2/first_server_assets/res/
  INFO root [2019-03-02 14:45:57,880] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine-2.4.2/kbe/bin/server/
  INFO root [2019-03-02 14:45:57,895] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.0.148 ( 192.168.0.148:14059 )
  INFO root [2019-03-02 14:45:57,897] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.0.148:14059, SOMAXCONN=5.
  INFO root [2019-03-02 14:45:57,897] - server-configs:
  INFO root [2019-03-02 14:45:57,898] - 	internalTcpAddr : 192.168.0.148:14059
  INFO root [2019-03-02 14:45:57,898] - 	componentID : 6000
  WARN root [2019-03-02 14:45:57,924] - invalid UID(0) <= 0, please check UID for environment! automatically set to 34643.
  INFO root [2019-03-02 14:45:59,436] - ---- cellappmgr Version: 2.4.2. ScriptVersion: 0.1.0. Pythoncore: 3.7.2. Protocol: ACC6C099AA23569403DEFB7E57FC06C6. Config: Debug X86. Built: 17:19:52 Feb 27 2019. AppID: 6000. UID: 34643. PID: 15628 ----
  INFO root [2019-03-02 14:45:59,437] - Server 19: Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz with 2097151kB RAM
  INFO root [2019-03-02 14:45:59,439] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-03-02 14:45:59,440] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-03-02 14:46:00,291] - ---- cellappmgr is running ----
 DEBUG root [2019-03-02 14:46:00,293] - Components::process(): Request for the process of identity...
  INFO root [2019-03-02 14:46:07,561] - Components::findComponents: find logger(1)...
  INFO root [2019-03-02 14:46:07,598] - Components::findComponents: found logger, addr:192.168.0.148:14064
  INFO root [2019-03-02 14:46:07,598] - Components::addComponent: reset orderLog, uid=34643!
  INFO root [2019-03-02 14:46:07,599] - Components::addComponent[logger], uid=34643, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 14:46:07,600] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-03-02 14:46:07,704] - Components::findComponents: found dbmgr, addr:192.168.0.148:14062
  INFO root [2019-03-02 14:46:07,704] - Components::addComponent: reset orderLog, uid=34643!
  INFO root [2019-03-02 14:46:07,704] - Components::addComponent[dbmgr], uid=34643, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-03-02 14:46:07,704] - Components::findComponents: found dbmgr, addr:192.168.0.148:14062
  WARN root [2019-03-02 14:46:08,705] - Forwarding logs to logger[192.168.0.148:14064/0/0/0]...
 ERROR root [2019-03-02 14:49:22,829] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! TCPPacketReceiver::processRecv(): error=10054

